### ISSUE TITLE
Automating use of resources

### ISSUE THREAD

**Issue by orthecreedence** [basisproject/tracker#126]
For some things, usage and consumption is a given. This goes for things like a widget being consumed to build a chair. Use is a function of the economic fabric and the productive system.

However, for things like housing, commercial and industrial property, land, raw materials, etc usage is a much more complex and intricate issue. Finding a way to limit this complexity and automate it will lower administrative costs and increase trust.

Effectively, this means some kind of smart contract system. I don't have any concrete ideas, so this is going to be a discussion issue. Note that this is an extension of part of #86.

**Comment by kaichaosun** [comment]
I'm still getting myself familiar with socialism, this may sound naive.
I think the usage of resource, means the value extracts from resource, either with frequency (like wheat as resource can use only once and 100% decrease of value of this usage) or time (like housing, 3% decrease of value of per-year usage).

**Comment by orthecreedence** [comment]
Yes, you're right. Some resources are used/consumed once, some are used over time. The goal is that for resources used over time, the stewards of that resource can decide how the cost is passed to the user. For instance, a house might have a yearly maintenance cost. It might make sense to have the person who lives in that house pay the cost of maintenance. On the other hand, people might want the cost of maintaining a public park be spread over everyone in the community, regardless of who uses it and when. The system should allow for both, and also in-between: for instance half the cost is covered by community, half is covered by the person or people in-use.
